Needs some strdup fixes for Windows:

```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\vcruntime.h:245:47: note: expanded from macro '_CRT_DEPRECATE_TEXT'
#define _CRT_DEPRECATE_TEXT(_Text) __declspec(deprecated(_Text))
                                              ^
isl_stream.c:655:15: warning: 'strdup' is deprecated: The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details. [-Wdeprecated-declarations]
                        tok->u.s = strdup("!=");
                                   ^
C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\ucrt\string.h:535:20: note: 'strdup' has been explicitly marked deprecated here
    _Check_return_ _CRT_NONSTDC_DEPRECATE(_strdup)
                   ^
[...]
lld-link: error: undefined symbol: __declspec(dllimport) strdup
>>> referenced by isl.lib(isl_ast.o):(isl_ast_expr_op_type_set_print_name)
>>> referenced by isl.lib(isl_input.o):(read_aff_domain)
>>> referenced by isl.lib(isl_input.o):(read_tuple_space)
>>> referenced 40 more times
```

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -4,7 +4,7 @@ if [[ "$target_platform" == "win-64" ]]; then
   export CFLAGS="$CFLAGS -O3 -Dstrdup=_strdup"
   export ac_cv_have_decl__BitScanForward=yes
   autoreconf -iv
-  ./configure --prefix=$PREFIX --with-int=imath-32 --disable-shared || (cat config.log && false)
+  ./configure --prefix=$PREFIX --with-int=imath-32 --disable-shared CFLAGS="$CFLAGS" || (cat config.log && false)
   patch_libtool
 else
   # Get an updated config.sub and config.guess

